VERSION

```diff
@@ -1 +1 @@
-2.0.0
+2.0.1
```

Comment:
Could you change this to something else? I think during the 1.0 release cycle we used `1.60.0` or or similar